sashimi-webapp/src/logic/filemanager/file.js

```diff
@@ -1,21 +1,53 @@
-import storage from '../../database/storage';
+import storage from 'src/database/storage';
 import Folder from './folder';
 
+/* Constants */
+
+// const RENAME_ERROR_MSG = `Another file in "${this.parentFolder.path}" has the same file name`;
+const RENAME_ERROR_MSG = 'Another file in the current folder has the same name';
+const MOVE_SAME_FOLDER_ERROR_MSG = 'Attempting to move to current folder';
+const MOVE_INVALID_FODLER_ERROR_MSG = 'Attempting to move to an invalid folder';
+
+/* Private Functions */
+
+function isCurrentFolder(destFolder) {
+  return destFolder.id === this.parentFolder.id;
+}
+
+function isInvalidFolder(destFolder) {
```

Comment:
Also, it could be cleaner to just 
`return Folder.getFolder(destFolder.id) == null`

Since you are doing the same thing for `function isCurrentFolder(destFolder) {` too